docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -103,7 +175,253 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
```

Comment:
There seems to be a missing example of using multiple supplemental files.

Reply:
OK, I think I've got it with this version, including emphasizing the precedence of overriding - thanks for keeping on me to improve the example!